web-components/src/components/buttons/ContainedButton.tsx

```diff
@@ -1,43 +1,28 @@
 import React from 'react';
-import { withStyles, DefaultTheme as Theme } from '@mui/styles';
 import Button, { ButtonProps } from '@mui/material/Button';
+import { styled } from '@mui/material';
 
 interface ContainedButtonProps extends ButtonProps {
   target?: string;
   rel?: string;
 }
 
-const CustomButton = withStyles((theme: Theme) => ({
-  root: {
-    borderRadius: '2px',
-    fontFamily: theme.typography.h1.fontFamily,
-    fontWeight: 800,
-    color: theme.palette.primary.main,
-    letterSpacing: '1px',
-    boxShadow: 'none',
-    whiteSpace: 'nowrap',
-    [theme.breakpoints.down('sm')]: {
```

Comment:
most of these are moved to the `muiTheme.ts`'s `component` section `styleOverrides`